schematic/manifest/generator.py

```diff
@@ -1254,18 +1256,19 @@ def get_empty_manifest(self, json_schema_filepath=None):
         )
         return manifest_url
 
-    def _get_missing_columns(self, wb_header, manifest_df):
-        # get headers from sheet 
-        manifest_df_header = manifest_df.columns
-
-        # find missing columns present in existing manifest but missing in latest schema
-        out_of_schema_columns = set(manifest_df_header) - set(wb_header)
-
-        return out_of_schema_columns
+    def _get_missing_columns(self, headers_1:list , headers_2:list) -> list:
+        """Compare two colunm sets and get cols that are in headers_1, but not headers_2
+        Args:
+            headers_1 (list): list of column headers
+            headers_2 (list): list of column headers
+        Returns: 
+            list: column headers in headers_1 but not headers_2
```

Comment:
will this truly return a list? I did a simple experiment: 
```
x = set(["apple", "banana", "cherry"])
y = set(["google", "microsoft", "apple"])

z = x-y

print(z)
```
Here, i am seeing: `{'cherry', 'banana'}` which is a set

Reply:
Yep you're right I can change the docs.